src/FeatureIntroControl/index.tsx

```diff
@@ -0,0 +1,213 @@
+  /**
+   * Override the default width
+   */
+  width?: string;
+  /**
+   * Override the default height
+   */
+  height?: string;
+}
+
+const FeatureIntroControl: React.FC<FeatureIntroControlProps> = ({
+  imagePlacement = "right",
+  width,
+  height,
+  className,
+  containerAs = "div",
+  learnMoreLinkAs = "a",
+  children,
+  ...props
+}) => {
+  const {
+    heading,
+    content,
+    image,
+    learnMoreLink,
+    dismissButton,
+  } = useFeatureIntroControlInternalContext();
+
+  /**
+   * Choose flex-direction based on imageDirection
+   */
+  const flexDirection = useMemo(() => {
+    switch (imagePlacement) {
+      case "bottom":
+        return "column-reverse";
+      case "top":
+        return "column";
+      case "right":
+        return "row";
+      default:
+        // left
+        return "row-reverse";
+    }
+  }, [imagePlacement]);
+
+  /**
+   * The width and height calculations for this component depend on whether or not there is an image,
+   * and whether or not there was a prop passed for width & height.
+   *
+   * By default, if an image is added the width / height is doubled respectively
+   * depending on whether the image direction is horizontal (left / right) or vertical(top / bottom)
+   *
+   * If a width or height is inputted, that height / width is not changed with an image
+   **/
+  const { finalWidth, finalHeight } = useMemo(() => {
+    const [baseWidth, baseHeight] = [248, 180];
+    const horizontalImage =
+      image && (imagePlacement === "right" || imagePlacement === "left");
+    const verticalImage =
+      image && (imagePlacement === "top" || imagePlacement === "bottom");
+    const defaultWidth = horizontalImage ? baseWidth * 2 : baseWidth;
+    const defaultHeight = verticalImage ? baseHeight * 2 : baseHeight;
+
+    return {
+      finalWidth: width || defaultWidth,
+      finalHeight: height || defaultHeight,
+    };
+  }, [height, image, imagePlacement, width]);
+
+  return (
+    <ClassNames>
+      {({ css, cx }) => {
+        return React.cloneElement(
+          createElementFromAs(containerAs),
+          {
+            ...props,
+            className: cx(
+              css({
+                width: finalWidth,
+                height: finalHeight,
+              }),
+              className,
+              React.isValidElement(containerAs) && containerAs.props.className,
+            ),
+            style: {
+              ...props.style,
+              ...(React.isValidElement(containerAs) && containerAs.props.style),
+            },
+          },
+          <div
+            css={{
+              backgroundColor: colors.white,
+              color: colors.grey.darker,
+              borderColor: colors.white,
+              borderRadius: 4,
+              display: "flex",
+              flexDirection,
+            }}
+          >
+            {React.isValidElement(image) &&
+              React.cloneElement(image, {
+                ...image.props,
+              })}
```

Comment:
I believe the cloning here isn't necessary 
```suggestion
            {image}
```